src/shared/functions.ts

```diff
@@ -25,29 +33,39 @@ export const getKeyFromObject = (element: RemoteChangeElement | ChangeResult): s
 
 export const supportsPartialDelete = (cmp: SourceComponent): boolean => !!cmp.type.supportsPartialDelete;
 
-export const isLwcLocalOnlyTest = (filePath: string): boolean =>
-  filePath.includes('__utam__') || filePath.includes('__tests__');
+export const excludeLwcLocalOnlyTest = (filePath: string): boolean =>
```

Comment:
I just inverted it for clearer use as a filter.  Both uses were negated by the consumer